.github/workflows/build.yaml

```diff
@@ -292,14 +373,23 @@ jobs:
       if: ${{ env.SSH_PRIVATE_KEY == '' }}
       run: git clone https://gitlab.com/OpenMW/openmw-deps.git
 
-    - name: Generate files containing a download url and hash
+    - name: Generate files containing a download url and hash for windows
       run: |
-        for file in vcpkg-x64-windows-*-${{ env.file }}.7z; do
-          filename=`echo "$file" | sed -r 's/vcpkg-x64-windows-(.+)-${{ env.file }}.7z/vcpkg-x64-\1-${{ env.file }}.txt/'`
-          echo "${{ github.server_url }}/${{ github.repository }}/releases/download/${{ env.file }}/$file" > "openmw-deps/windows/$filename"
-          shasum -a 512 "$file" >> "openmw-deps/windows/$filename"
+        mkdir -p openmw-deps/windows
+        for src in vcpkg-x64-windows-*-${{ env.file }}.7z; do
+          dst=$(echo "${src:?}" | sed -r 's/vcpkg-x64-windows-(.+)-${{ env.file }}.7z/vcpkg-x64-\1-${{ env.file }}.txt/')
+          echo "${{ github.server_url }}/${{ github.repository }}/releases/download/${{ env.file }}/${src:?}" > "openmw-deps/windows/${dst:?}"
+          shasum -a 512 "${src:?}" >> "openmw-deps/windows/${dst:?}"
         done
 
+    - name: Generate files containing a download url and hash for linux
+      run: |
+        mkdir -p openmw-deps/linux
+        src=vcpkg-x64-linux-${{ env.file }}.7z
```

Comment:
It might be sensible to include the compiler and version in the name like we do for Windows as it affects the ABI, just like it does on Windows. Thinking about it, it might have made some sense to put `msvc2022` in the Windows filenames instead of just `2022`, but it's not as big a deal as for Clang and GCC as 2022 won't get confused with 20 or 14, but it's not as obvious that 14 is a GCC version rather than a Clang one.